vendor/engines/secure_rooms/Rakefile

```diff
@@ -0,0 +1,37 @@
+begin
+  require 'bundler/setup'
```

Comment:
Presume this was autogenerated due to style? Also is this set up for minitest instead of rspec?